share/runtime-cleanup.tmpl

```diff
@@ -344,6 +344,9 @@ removefrom redhat-logos /usr/share/{plymouth,backgrounds}*
 removefrom subscription-manager --allbut /etc/rhsm/* /usr/share/rhsm/* /var/log/rhsm/* \
     /etc/yum/pluginconf.d/product-id.conf /usr/lib/yum-plugins/product-id.py
 
+## OSCAP cleanups
+removefrom scap-security-guide --allbut /usr/share/xml/scap/ssg/content/ssg-rhel7-ds.xml
+
```

Comment:
Could add:

removefrom scap-security-guide /usr/share/scap-security-guide/

as well.

Reply:
That already removes those files as well. --allbut removes every file in the packages except those matching the argument glob.